snowexsql/tables/campaign_observation.py

```diff
@@ -20,13 +20,18 @@ class CampaignObservation(
     __tablename__ = 'campaign_observations'
 
     # Data columns
-    name = Column(Text)
+    name = Column(Text, nullable=False)
```

Comment:
Are we going to have a name for campaign observations? I'm thinking magnaprobe, GPR returns, etc might note have a name per row

Reply:
Could those be the same as the site names those profiles are taken?
From my experience, those are usually with a pit or close to one.

Reply:
I like that logic!